api/index.js

```diff
@@ -8,6 +9,31 @@
    res.send('Hello World HERE!')
  })
 
+// post endpoint
+router.post('/testing', [//validations
+  body('name', 'Name is mandatory').trim().escape().not().isEmpty(),
+  body('email', 'Invalid email').trim().escape().isEmail(),
+  body('date', 'Invalid date (format YYYY-MM-DD)').trim().escape().isISO8601()
+], (req, res) => {
+
+  	const errors = validationResult(req);
+  	if (!errors.isEmpty()) {
+	  return res.status(422).json({ errors: errors.array() });
+	}
+
+	var body 	= req.body;
```

Comment:
Usar `let` en lugar de `var`.
https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var